app/src/sharedTest/java/org/oppia/app/story/StoryFragmentTest.kt

```diff
@@ -80,6 +82,30 @@ class StoryFragmentTest {
     }
   }
 
+  @Test
+  fun testStoryFragment_changeConfiguration_textViewIsShownCorrectly() {
+    launch<StoryFragmentTestActivity>(createTestActivityIntent(TEST_STORY_ID_1)).use {
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.story_chapter_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(0)).check(
+        matches(
+          withText("Second Story")
```

Comment:
This test case is still failing. 
<img width="1314" alt="Screenshot 2020-02-05 at 9 59 44 AM" src="https://user-images.githubusercontent.com/9396084/73873163-12af2e00-4877-11ea-9d4e-7f507ec570ec.png">